shared/metadata_test.go

```diff
@@ -32,6 +31,7 @@ links:
       status: FAIL
     - test: c.html
   - url: https://github-issue.com/1234
+    label: labelB
```

Comment:
Here there's both `url` and `label`. I guess that makes sense when it applies to a single test, but is it also possible to split these into two items? If it is possible, I wonder if it wouldn't be easier overall to *only* allow the split form, in other words disallowing having both `url` and `label` at the same time?

Reply:
Per discussion with @KyleJu this is just internal to the wpt.fyi representation, in wpt-metadata it will be allowed to have either `url` or `label`, or both. And we'll have lints to catch the situation where there's redundant entries. In that case, allowing both forms is fine.